k8s/recommended.yaml

```diff
@@ -0,0 +1,51 @@
+### Yatai Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  generation: 1
+  labels:
+    app: yatai-service
+  name: yatai-service
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: yatai-service
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: yatai-service
+    spec:
+      containers:
+      - image: bentoml/yatai-service:0.7.2
```

Comment:
I think we released new version of yatai-service since.  What do you think about pull the `latest` as default and give the user option to override the version?